.github/scripts/config.sh

```diff
@@ -80,4 +80,11 @@ PATH="$PATH:$TOOLCHAIN_PATH/bin"
 if [[ "$CCACHE" = 1 ]]; then
     PATH=$CCACHE_LIB_DIR:$TOOLCHAIN_CCACHE_LIB_DIR:$PATH
     export CCACHE_DIR=$CCACHE_DIR_PATH
+    set +o pipefail
```

Comment:
Invoking a sub-shell is safer and there is no need to restore the setting:
```bash
$ (set -o pipefail && command1 | command2 | command3)
```

Reply:
Yes, it was my initial idea. Somehow I decided to do in different way. I will change it.

Reply:
Not needed anymore after refactoring.